configs/recognition/c3d/c3d_sports1m_16x1x1_30e_ucf101_rgb.py

```diff
@@ -0,0 +1,134 @@
+# model settings
+model = dict(
+    type='Recognizer2D',
+    backbone=dict(
+        type='C3D',
+        pretrained='https://download.openmmlab.com/mmaction/recognition/c3d'
+        '/c3d_sports1m_pretrain_20201016-dcc47ddc.pth',
```

Comment:
you can use `# noqa: E501` like [this](https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/csn/ircsn_ig65m_pretrained_bnfrozen_r152_32x2x1_58e_kinetics400_rgb.py#L7-L8)